modules/gapi/src/api/gbackend_priv.hpp

```diff
@@ -50,14 +50,22 @@ class GAPI_EXPORTS cv::gapi::GBackend::Priv
                          const GCompileArgs &args,
                          const std::vector<ade::NodeHandle> &nodes) const;
 
+
     virtual EPtr compile(const ade::Graph   &graph,
                          const GCompileArgs &args,
                          const std::vector<ade::NodeHandle> &nodes,
                          const std::vector<cv::gimpl::Data>& ins_data,
                          const std::vector<cv::gimpl::Data>& outs_data) const;
 
+    // Ask backend to provide general backend-specific compiler passes
     virtual void addBackendPasses(ade::ExecutionEngineSetupContext &);
 
+    // Ask compiler to put extra meta-sensitive backend passes Since
```

Comment:
That's my mistake here, `compiler` -> `backend`. Could you please fix it?

Reply:
fixed